packages/sdk-codegen-scripts/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+### [21.0.11](https://www.github.com/looker-open-source/sdk-codegen/compare/sdk-codegen-scripts-v21.0.10...sdk-codegen-scripts-v21.0.11) (2021-03-30)
+
```

Comment:
Hah! the `--versions` comment would be relevant here. This is probably due to me having to touch too many other files in other packages?